src/components/SearchBox/index.jsx

```diff
@@ -38,38 +39,65 @@ const dropIn = {
   },
 };
 
+const client = useAxios();
+function searchGame({ queryKey }) {
+  const searchString = queryKey[1];
+  return client.get("/games", {
+    params: {
+      populate: "*",
+      filters: {
+        name: {
+          $containsi: searchString,
+        },
+      },
+    },
+  });
+}
+
+function sanitizeData(data) {
```

Comment:
I saw this function is being used in some other component as well, we can abstract this.